### PR TITLE
Feature/highlightjs

### LINKED ISSUE #213: Readd syntax highlighting in render output
**Describe the solution you'd like**

Codimd 1.6 uses Prism for syntax highlighting in the rendered markdown
image

**Additional context**

https://github.com/codimd/server/blob/20a1563d7f81d0454533c569b6505c71a720956f/public/js/extra.js#L488

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
The markdown renderer

### Description
This PR adds highlight js rendering

### Steps

- [x] added implementation
- [x] added / updated tests
- [x] added / updated documentation
- [x] extended changelog

### Related Issue(s)
Fixes #213

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
 - Speakerdeck embedding
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
+- Code highlighting for the follow languages
+    - tiddlywiki
+    - mediawiki
+    - jsx
```
Comment (DerMolly) [inline CHANGELOG.md:22]: Do we give some explanation? I mean I can live without, but maybe that would be a good idea?

File: src/components/editor/markdown-renderer/replace-components/highlighted-code/highlighted-code.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.markdown-body {
+  @import '../../../../../../node_modules/highlight.js/styles/github-gist' ;
+  code.hljs {
+  }
```
Comment (DerMolly) [inline src/components/editor/markdown-renderer/replace-components/highlighted-code/highlighted-code.scss:4]: what is this empty class for?

Comment (mrdrogdrog) [reply src/components/editor/markdown-renderer/replace-components/highlighted-code/highlighted-code.scss]: It's not used anymore.
I'll remove it.

File: CHANGELOG.md (hunk 1 above)
Comment (mrdrogdrog) [reply CHANGELOG.md]: Good point. 

Extended

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,6 +16,10 @@
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
 - Speakerdeck embedding
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
+- We are now using highlight.js instead of highlight.js+prism.js. The highlighting for the following languages has been dropped:
+    - tiddlywiki
+    - mediawiki
+    - jsx
```
Comment (ErikMichelson) [inline CHANGELOG.md:22]: I would add a space between highlight.js+prism.js (see suggestion above).
And maybe add how many new languages we support instead (if any).

File: src/components/editor/markdown-renderer/markdown-renderer.tsx [hunk 4]
```diff
@@ -10,12 +10,14 @@ import inserted from 'markdown-it-ins'
 import marked from 'markdown-it-mark'
 import markdownItRegex from 'markdown-it-regex'
 import subscript from 'markdown-it-sub'
+
```
Comment (ErikMichelson) [inline src/components/editor/markdown-renderer/markdown-renderer.tsx:13]: Any reason for the blank line?

Comment (mrdrogdrog) [reply src/components/editor/markdown-renderer/markdown-renderer.tsx]: No. I removed it.

File: CHANGELOG.md (hunk 3 above)
Comment (mrdrogdrog) [reply CHANGELOG.md]: better?

**Comment by mrdrogdrog** [comment]
Argh damn. I forgot the syntax for "start with line number xyz" and "continue with the line number form the last code block" -_-.... I'll cover that with another pr later.

**Pull request merged**